zjit/src/hir.rs

```diff
@@ -530,6 +535,7 @@ impl std::fmt::Display for Insn {
     }
 }
 
+/// A basic block in a [`Function`].
```

Comment:
We should mention that it's an extended basic block, which might otherwise people if they see multiple control flow operations in it